packages/contracts/test/framework/utils/token-factory.ts

```diff
@@ -51,7 +51,7 @@ interface MintConfig {
 
 const zeroAddr = ethers.constants.AddressZero;
 
-describe('Core: TokenFactory', () => {
+describe.skip('Core: TokenFactory', () => {
```

Comment:
Should we have this as `skipIfZkSync`

Reply:
No, it will make things hardhat as this also doesn't run anymore on hardhat due to smock library being removed. We anyways don't use this yet, so this can be skipped for now.